GVFS/GVFS.Platform.Mac/MacFileSystemVirtualizer.cs

```diff
@@ -428,6 +429,12 @@ private bool TryGetSymLinkTarget(string sha, out string symLinkTarget)
             }
 
             return Result.EIOError;
+        }
+
+        private Result OnLogError(string errorMessage)

```

Comment:
Minor: Return type can be `void` instead of `Result` as we're always returning `Success` and the caller isn't checking the return value.

Reply:
If you'd like, this change can be left for #395

Reply:
We might as well update it now, I'll do that.  We don't check the result and the tracer function we call is a void.